packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -646,6 +650,18 @@ export class MgtPerson extends MgtTemplatedComponent {
     `;
   }
 
+  private line1ClickedEvent() {
+    this.fireCustomEvent('line1clicked', this.personDetails);
+  }
+
+  private line2ClickedEvent() {
+    this.fireCustomEvent('line2clicked', this.personDetails);
+  }
+
+  private line3ClickedEvent() {
+    this.fireCustomEvent('line3clicked', this.personDetails);
+  }
+
```

Comment:
Could you rename these handlers to `handleLine1Clicked`, etc? This way it's more inline with the convention for naming event handlers.